receiver/postgresqlreceiver/metadata.yaml

```diff
@@ -297,3 +297,13 @@ metrics:
       value_type: int
     extended_documentation: |
       This metric requires WAL to be enabled with at least one replica.
+  postgresql.wal.delay:
+    attributes: [wal_operation_lag, replication_client]
+    description: Time between flushing recent WAL locally and receiving notification that the standby server has completed an operation with it captured in seconds and milliseconds.
+    enabled: true
```

Comment:
Unfortunately with new metrics we must make them optional, otherwise it is considered a breaking change.

Reply:
is switching `enabled` flag enough to make them optional?

Reply:
Yes! I should have been clear on that. That's all we need, thanks.